Gemfile.lock

```diff
@@ -311,69 +312,73 @@ GEM
       actionpack (>= 5.0)
       railties (>= 5.0)
     rexml (3.2.5)
-    rubocop (1.26.1)
+    rubocop (1.35.1)
+      json (~> 2.3)
       parallel (~> 1.10)
-      parser (>= 3.1.0.0)
+      parser (>= 3.1.2.1)
       rainbow (>= 2.2.2, < 4.0)
       regexp_parser (>= 1.8, < 3.0)
-      rexml
-      rubocop-ast (>= 1.16.0, < 2.0)
+      rexml (>= 3.2.5, < 4.0)
+      rubocop-ast (>= 1.20.1, < 2.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 3.0)
-    rubocop-ast (1.16.0)
+    rubocop-ast (1.21.0)
       parser (>= 3.1.1.0)
-    rubocop-performance (1.13.3)
+    rubocop-performance (1.14.3)
```

Comment:
### [rubocop-performance](https://docs.rubocop.org/rubocop-performance/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.13.3 → 1.14.3

[_[change-log](https://github.com/rubocop/rubocop-performance/blob/master/CHANGELOG.md), [source-code](https://github.com/rubocop/rubocop-performance/), [gem-diff](https://my.diffend.io/gems/rubocop-performance/1.13.3/1.14.3)_]





<details>
<summary>Commits</summary>

A change of **46** commits. See the full changes on [the compare page](https://github.com/rubocop/rubocop-performance/compare/v1.13.3...v1.14.3).

These are the first 10 commits:
- (9d7c8b2) [Switch back docs version to master](https://github.com/rubocop/rubocop-performance/commit/9d7c8b20430192957c3cd9fbe6ab87d7dc4d55ce)
- (219bf57) [Use `actions/checkout` version 3](https://github.com/rubocop/rubocop-performance/commit/219bf570999d291ef657de97792b493e43f42393)
- (5cc1576) [Tweak offense message for `Performance/Casecmp`](https://github.com/rubocop/rubocop-performance/commit/5cc157679b39803261402109b38f30034a6cb8e7)
- (f4fad16) [Suppress a RuboCop offense](https://github.com/rubocop/rubocop-performance/commit/f4fad1693b688dd7dc82b7dd0b29a617ba9834dc)
- (a1cfc94) [Drop Ruby 2.5 support](https://github.com/rubocop/rubocop-performance/commit/a1cfc94f631d1deb9481aca6445d1d313d1c47b2)
- (90a4f0a) [Update a link that make use of deprecated git.io](https://github.com/rubocop/rubocop-performance/commit/90a4f0a95f8433e2fb6b5ada346a62be9a612d05)
- (f272803) [Merge pull request #287 from koic/drop_ruby_2_5_support](https://github.com/rubocop/rubocop-performance/commit/f27280397777e60d120ffa313a2a87ed183bd116)
- (1e1a972) [Add consitency test for config defaults so that cops are sor…](https://github.com/rubocop/rubocop-performance/commit/1e1a972b0f5dea0756b29877ecb68c0ce4d42204)
- (076e07c) [Tweak a couple of changelog entries](https://github.com/rubocop/rubocop-performance/commit/076e07cfeeb9c3c919ad9284bda056ccd4345d5e)
- (a527faf) [Use `behavior` instead of `behaviour`](https://github.com/rubocop/rubocop-performance/commit/a527faf6c803de5cd2c8b3f2377f1a52ca824866)


</details>